scripts/fluentd-bigquery/Gemfile

```diff
@@ -0,0 +1,4 @@
+source "https://rubygems.org"
+ruby '~> 2.6.0'
+
+gem "fluent-plugin-bigquery", "~> 2.2.0"
```

Comment:
Likewise here, should we upgrade it?

Reply:
I was worried the newer plugin version may have some breaking changes but it seems to work well for the functionality we use. Done.